vanilla-framework/vanilla-framework
# Add the MAAS layout for testing

## Done
Add a layout which replicates the MAAS app layout for testing.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/templates/maas-layout/
- Check it looks like the design:

## Design
![mmas-vision__machine-listing](https://user-images.githubusercontent.com/1413534/37041257-afd67a78-2153-11e8-898d-b0c1f6b5069f.png)
